news.js

```diff
@@ -0,0 +1,214 @@
+//variable used to store the complete data of all news channels
+let news;
+window.onload = () => {
+	console.log('onload');
+	news = new News();
+};
+class News {
+	constructor() {
+		this.fullData = '';
+		this.data1 = [];
+		this.email = [];
+		this.displayPage();
+	}
+
+	//Function to display the selected news category
+	selectCategoryNews = () => {
+		let selectedCategory = document.getElementById('sel-category').value;
+		let display = ' ';
+		let indexDisplay = [];
+		let data = this.data1;
+
+		let selectedData = data.filter(function(e, index) {
+			if (selectedCategory === e.source.name) indexDisplay.push(index);
```

Comment:
rewrite as 
if (selectedCategory === e.source.name) {
indexDisplay.push(index);
}

Never write a if logic in one line.